Sources/Supporting/MessageKit+Availability.swift

```diff
@@ -24,6 +24,20 @@
 
 import Foundation
 
+// MARK: - Deprecated Protocols
+
+@available(*, deprecated: 0.11.0, message: "Now typealias for MessagesDisplayDelegate")
```

Comment:
Should these messages maybe say *why* it was deprecated + what the alternative is?

Reply:
@BasThomas Sorry I merged this quick because something else I'm doing depends on it.
I was hoping the message I wrote was clear, that it is now a `typealias` for `MessagesDisplayDelegate` 😭

Reply:
Sure thing, I can change the message before the release 👍

Reply:
I'll spin one up 👍